meteor/sofie-server/model/exceptions/rundown-active-exception.ts

```diff
@@ -0,0 +1,8 @@
+import { ErrorCode } from '../enums/error-code'
+import { Exception } from './exception'
+
+export class RundownActiveException extends Exception {
```

Comment:
Would be nice if this was called `RundownAlreadyActiveException`.

Reply:
Would be nice if we used the same "sentence" structure for naming. E.g. such that this becomes `ActiveRundownException` or some other common way of naming Exceptions (and other constructs).

Reply:
Have renamed the exception to `ActiveRundownException` The error code inside remains the same.

Reply:
We should ask the question: How specific do we want our Exceptions to be? We don't want to manage 100+ exceptions.

Maybe we should delete `AlreadyActiveException` and just use `ActiveRundownException` for that scenario as well?

Reply:
That might be the better way of doing it.
Having a message along the lines of `'Cannot activate an already active rundown'`

Reply:
Should that change be done in this PR, or at some other point?